.travis.yml

```diff
@@ -273,8 +275,8 @@ addons:
 # virtualenv is listed here explicitly to make sure it is upgraded when
 # CERTBOT_NO_PIN is set to work around failures we've seen when using an older
 # version of virtualenv.
-install: "tools/pip_install.py -U codecov tox virtualenv"
-script: tox
+install: 'tools/pip_install.py -U codecov tox virtualenv'
+script: '$TRAVIS_RETRY tox'
```

Comment:
nit: Can you add a brief comment here explaining that TRAVIS_RETRY is set for flakey tests and for most tests it is an empty string and has no effect?

Reply:
Done.